koji/flatten-el8/koji-ansible/roles/grobisplitter/files/splitter.py

```diff
@@ -0,0 +1,397 @@
+#!/bin/python3
```

Comment:
Not all distros symlink `/bin` to `/usr/bin`
```suggestion
#!/usr/bin/python3
```

Reply:
. I'd prefer not to deviate from their script more than necessary at this time.